sys/include/net/gnrc/netif/hdr.h

```diff
@@ -199,6 +199,33 @@ gnrc_pktsnip_t *gnrc_netif_hdr_build(uint8_t *src, uint8_t src_len, uint8_t *dst
  */
 void gnrc_netif_hdr_print(gnrc_netif_hdr_t *hdr);
 
+/* @brief   Fetch the netif header flags of a gnrc packet
+ *
+ * @param[in]   pkt     gnrc packet from whom to fetch
+ * @param[out]  flags   pointer to flags will be stored here
+ *
+ * @return              the result of fetching flags of @p pkt
+ */
+int gnrc_netif_hdr_get_flag(gnrc_pktsnip_t* pkt, uint8_t* flags);
```

Comment:
Sorry for the carelessness, and do you against the current design of 
`int gnrc_netif_hdr_get_flag(gnrc_pktsnip_t* pkt, uint8_t* flags);` ? which will return `-ENODEV` in case the packet is lack of `netif header snip`